tests/spark/autologging/datasource/test_spark_datasource_autologging_unit.py

```diff
@@ -7,14 +7,6 @@
 from mlflow.exceptions import MlflowException
 from mlflow.spark.autologging import PythonSubscriber, _get_current_listener
 
-from tests.spark.autologging.utils import _get_or_create_spark_session
-
-
-@pytest.fixture(scope="module")
-def spark_session():
-    with _get_or_create_spark_session() as session:
-        yield session
```

Comment:
This is redundant. `tests/spark/autologging/datasource/conftest.py` has the exact same fixture.